qa/tasks/ceph_deploy.py

```diff
@@ -46,7 +46,7 @@ def download_ceph_deploy(ctx, config):
         system_type = teuthology.get_system_type(ceph_admin)
 
         if system_type == 'rpm':
-            package = 'python34' if py_ver == '3' else 'python'
+            package = 'python36' if py_ver == '3' else 'python'
             ctx.cluster.run(args=[
                 'sudo', 'yum', '-y', 'install',
                 package, 'python-virtualenv'
```

Comment:
I am also wondering about `python-virtualenv`, here. What do we use it for? If we want to create python3 virtualenv then we should also version that package. Otherwise, we would be getting a python2 virtualenv.

Reply:
the only test i found using the venv is `qa/tasks/cram.py` and `qa/tasks/tox.py`.  i don't think they need python3.

more importantly, this change does not break them.

Reply:
True, I was just wondering whether we need to change that, too.

Reply:
yeah, we should definitely change them once Centos/RHEL 7 are moving over to python3.